### DISCUSSION TITLE
Use DHT to map challenge, y, to IP address of machine with the plot data

### DISCUSSION THREAD

**Discussion by pointlander** [Chia-Network/chia-blockchain#4641] [Ideas]
It will be possible to look up in a distributed hash table the challenge value and get an IP address back. The block is then sent to that IP address for proving. This will make Chia more power efficient and be a step in the direction of a fully distributed block chain, but I currently don't know how to solve that problem.

**Comment by ilyakooo0** [comment]
1. I'm not sure I want my IP address in a DHT
2. IP addresses can easily change

**Reply by pointlander** [reply]
CJDNS could be used: https://github.com/cjdelisle/cjdns
